check-url

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+URL="$1"
+
+if [[ $URL == ^http://* ]]; then
```

Comment:
It doesn't, I forgot to invert the assertion and it seems useless using `curl`... :disappointed: 

My bad, thanks :wink: